gmail-to-salesforce-lead/main.bal

```diff
@@ -31,9 +31,9 @@ configurable string salesforceAccessToken = ?;
 
 const LABEL = "Lead";
 
-final gmail:Client gmailClient = check new ({auth: {token: gmailAccessToken}});
-final openAI:Client openAIClient = check new ({auth: {token: openAIKey}});
-final sfdc:Client sfdcClient = check new ({baseUrl: salesforceBaseUrl, auth: {token: salesforceAccessToken}});
+final gmail:Client gmail = check new ({auth: {token: gmailAccessToken}});
+final openAI:Client openAI = check new ({auth: {token: openAIKey}});
```

Comment:
Incorrect to use alias as `openAI` as per our naming conventions. It should be `openAi`. https://learn-ballerina.github.io/best_practices/naming_conventions.html#names-with-abbrevations
Btw, do we really need an alias here? IMO, having the following is fine.

```ballerina
import ballerinax/openai.chat;

final chat:Client openAiChat = check new ({auth: {token: "openAIKey"}});
```

Reply:
updated in [c822b14](https://github.com/ballerina-guides/integration-samples/pull/38/commits/c822b142996d293e1fb33d5ae48e52ea6dfda462)